src/utils/shuffle.ts

```diff
@@ -0,0 +1,12 @@
+export default function shuffleArray(array) {
+  var i = 0,
+    j = 0,
+    temp = null
+
+  for (i = array.length - 1; i > 0; i -= 1) {
+    j = Math.floor(Math.random() * (i + 1))
+    temp = array[i]
+    array[i] = array[j]
+    array[j] = temp
+  }
+}
```

Comment:
We should consider installing lodash. Between this and the debounce utility function, it might be useful.

Reply:
Yeah you're right.